spec/factories/tasks.rb

```diff
@@ -5,5 +5,10 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    trait :done_yesterday do
+      status { 'done' }
```

Comment:
enumの値はSymbolで定義しましょう。
```suggestion
      status { :done }
```